### PR TITLE
Added N64 Mouse support in NRage

### PR DESCRIPTION
**Pull request by LuigiBlood**

### CODE REVIEW

File: Source/nragev20/NRagePluginV2.rc [hunk 1]
```diff
@@ -524,25 +524,29 @@ END
...
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION        VER_FILE_VERSION
- PRODUCTVERSION     VER_PRODUCT_VERSION
- FILEFLAGSMASK      0x3fL
- FILEFLAGS          VER_FILEFLAGS
- FILEOS             VER_FILEOS
- FILETYPE           VER_FILETYPE
- FILESUBTYPE        0x0L
+ FILEVERSION 2,4,0,4
```
Comment (project64) [inline Source/nragev20/NRagePluginV2.rc]: any particular reason changing the version structure has to do with the mouse?

**Comment by LuigiBlood** [comment]
That might have been VS2013 changing everything when I added a new checkbox maybe...?
I didn't really see that at first.

**Comment by project64** [comment]
I find I check the difference before checking in and revert the changes VS sometimes makes so it is only the relevant changes I need.

**Comment by LuigiBlood** [comment]
I should have fixed the problem now. I used the original file and changed only the lines that should be changed.

**Pull request merged**